homeassistant/components/suez_water/quality_scale.yaml

```diff
@@ -0,0 +1,88 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data:
+    status: todo
+    comment: coordinator is created during setup, should be stored in runtime_data
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: no subscription to api
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: no service action
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  docs-actions:
+    status: exempt
+    comment: no service action
+  brands: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
```

Comment:
This looks like it may log more than once when unavailable given the coordinator has its own exception logging.